acrobat/blocks/dc-converter-widget/dc-converter-widget.js

```diff
@@ -144,6 +151,13 @@ export default function init(element) {
   let REDIRECT_URL = '';
   let DC_GENERATE_CACHE_URL = '';
 
+  createTag.then((createTag) => {
```

Comment:
this is too late to prefetch. we need the prefetch on the initial HTML document.